util/settings/settings.go

```diff
@@ -406,6 +410,8 @@ const (
 	settingServerPrivateKey = "tls.key"
 	// settingURLKey designates the key where Argo CD's external URL is set
 	settingURLKey = "url"
+	// settingURLsKey designates the key where Argo CD's external URLs is set
+	settingURLsKey = "urls"
```

Comment:
Invalid key to load configuration

```suggestion
	// settingAdditionalUrlsKey designates the key where Argo CD's additional external URLs are set
	settingAdditionalUrlsKey = "additionalUrls"
```